episodes/2-keras.Rmd

```diff
@@ -489,11 +489,30 @@ Looking at the training curve we have just made.
    * Does the graph look very jittery?
 2. Do you think the resulting trained network will work well on the test set?
 
+When the training process does not go well:
+
+3. (optional) Something went wrong here during training. What could be the problem, and how do you see that in the training curve?
+Also compare the range on the y-axis with the previous training curve.
+![](../fig/02_bad_training_history_1.png){alt='Very jittery training curve. The loss does not really decrease'}
```

Comment:
I dunno, I think the loss definitely does decrease, not as much though. Would it help to run this same thing with more iterations so you can see that it stagnates for a longer time?

Reply:
I agree that the trend of the curve shows some decrease with epochs. I think you could resolve this by adjusting the alt-text to describe the range on the y-axis: this would also be consistent with the sentence above ("Also compare the range on the y-axis with the previous training curve.").

Reply:
Good points! I resolved it by having a much more elaborate alt text, indeed referring to the fact that the y-axis range is different.